### PR TITLE
updating Mailjet code samples for Send API v3.1

### PR DESCRIPTION
**Pull request by adamyanliev**
Mailjet Send API v3.1 is being released, so updating the code samples to match the new Send API v3.1 configuration. Reference can be found [here](https://dev.mailjet.com/beta/).

### CODE REVIEW

**Comment by lesv** [comment]
Thanks for the contribution.

Shouldn't you also update the pom.xml's for this.

**Comment by adamyanliev** [comment]
@lesv checked the pom.xml files, they are up-to-date.

File: compute/mailjet/src/main/java/com/example/compute/mailjet/MailjetSender.java [hunk 1]
```diff
@@ -34,23 +35,27 @@
...
-    MailjetRequest email = new MailjetRequest(Email.resource)
-        .property(Email.FROMEMAIL, sender)
-        .property(Email.FROMNAME, "pandora")
-        .property(Email.SUBJECT, "Your email flight plan!")
-        .property(Email.TEXTPART,
-            "Dear passenger, welcome to Mailjet! May the delivery force be with you!")
-        .property(Email.HTMLPART,
-            "<h3>Dear passenger, welcome to Mailjet!</h3><br/>May the delivery force be with you!")
-        .property(Email.RECIPIENTS, new JSONArray().put(new JSONObject().put("Email", recipient)));
+    MailjetRequest email = new MailjetRequest(Emailv31.resource)
```
Comment (lesv) [inline compute/mailjet/src/main/java/com/example/compute/mailjet/MailjetSender.java]: We don't use tabs, but do use two spaces, not 4.  If you could fix that, then I'll LGTM

**Comment by adamyanliev** [comment]
@lesv thanks for the explanation. I updated the spacing, should be fine now.

**Comment by lesv** [comment]
Thank you again for your contribution.

**Pull request merged**